### PR TITLE
[Issue 442] refactored the documentation page a bit

### PR DESCRIPTION
**Pull request by vikv1**
Moved the unit and regression test subsections under the Developer section
Moved student quickstart under user documentation

image

### CODE REVIEW

**Comment by stiber** [comment]
Looks like things are shaping up nicely! When you feel like this is ready for prime time, let's solicit feedback from the lab members.

File: docs/Developer/UnitTests.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+## Unit Tests
+
+We use [Googletest](GoogleTestsTutorial.md) to develop our unit tests.
+
+
+### Running The Unit And Regression tests
+Unit and regression tests are important to minimize the chances that changes to the code base will produce undesired side effects. 
+
+We have a battery of tests that are run by a GitHub action on any `push` or `pull request` against the master branch. These tests are only executed for the CPU implementation of Graphitti.
```
Comment (stiber) [inline docs/Developer/UnitTests.md:13]: Now, these are also run against the development branch.

**Comment by vikv1** [comment]
Fixes Issue #442

**Comment by vikv1** [comment]
going to make new pr

**Pull request closed without merging**